src/tic_tac_toe/computer.clj

```diff
@@ -0,0 +1,24 @@
+(ns tic-tac-toe.computer
+  (:require [tic-tac-toe.announcer :as announcer]
+            [tic-tac-toe.player :as player]
+            [tic-tac-toe.referee :as referee]
+            [tic-tac-toe.user-interface :as ui]))
+
+(defn- first-available
+  [board]
+  (loop [i 0
+         board board]
+  (if (= (board i) " ")
+    i
+    (recur (inc i) board))))
+
+(defrecord Computer []
+  player/Player
+  (pick-move
+    [this board]
+    (Thread/sleep 2000)
```

Comment:
Does this effect how fast your tests run?

Reply:
It does (and it was bugging me), but you asking made me realize I can abstract it into the IO as an output pause.  I may also get rid of it later depending on how the flow of the game changes.

Reply:
Moved this into [IO protocol](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-6fc8428f8d5a98bc15e079cfdc7c2346R8).  I will leave it there for now and see if it's needed or not with minimax.